spec/redis_spec.cr

```diff
@@ -1256,7 +1259,7 @@ describe Redis do
     it "#pexpireat" do
       redis.set("temp", "3")
       redis.pexpireat("temp", 1555555555005).should eq(1)
-      redis.pttl("temp").should be > 2990
+      redis.pttl("temp").should eq(-2)
```

Comment:
I think a better fix is to push the update the `pexpireat` timestamp far into the future.